server/routes/probationPractitionerReferrals/probationPractitionerReferralsController.test.ts

```diff
@@ -132,9 +134,10 @@ describe('GET /probation-practitioner/dashboard', () => {
           },
         }),
       ]
+      const page = pageFactory.pageContent(referrals).build() as Page<SentReferral>
 
       interventionsService.getIntervention.mockResolvedValue(intervention)
-      interventionsService.getSentReferralsForUserToken.mockResolvedValue(referrals)
+      interventionsService.getSentReferralsForUserTokenPaged.mockResolvedValue(page)
```

Comment:
Is it worth testing that the pagination parameter (`page`) is correct passed through, with edge cases like missing = default value, string = what happens, etc?